### ISSUE TITLE
Zotero2Srophé

### ISSUE THREAD

**Issue by mMoliere** [majlis-erc/srophe#130]
@wsalesky I want to convert the additions to the Zotero folder, but I'm having the same problem as last time. The update breaks down after 650 entries. Last time, I believe, you managed to update. Can you explain to me what you did?

**Comment by wsalesky** [comment]
@mMoliere Does it time out? If not I suspect I paged through, and skipped the problematic entries. 
I will send you an email on how to page through manually.

**Comment by wsalesky** [comment]
@mMoliere Actually, it may be a permissions error. Let me update the permissions and try to run it again.

**Comment by wsalesky** [comment]
@mMoliere I ran the update... does it look correct to you? I am suspicious that it truly updated as it ran much faster than normal.

**Comment by mMoliere** [comment]
@wsalesky I can't detect changes in the Github data repository, so I think it didn't work.

**Comment by wsalesky** [comment]
@mMoliere Can you check for changes now?

**Comment by mMoliere** [comment]
@wsalesky No, still no changes

**Comment by mMoliere** [comment]
@wsalesky After you added the full display of bibliographic records to JALit-Srophé we noticed that certain data fields are missing: "Encyclopedia Title", "Pages" and "DOI". Looking at the XML files generated by Zotero2Bibl I noticed that they had not been imported into dedicated fields. (They do appear though in the formatted `bibl` element https://github.com/majlis-erc/majlis-data/blob/3e396c0a10ec3a1f101412440f179ebfa2e9160a/data/bibl/tei/2AHY89CU.xml#L78)

What can we do about it? Is it advisable to modify Zotero2Bibl to include these fields and then reimport everything?

To us it would also be acceptable if Srophé displayed the formatted `bibl` element.

**Comment by wsalesky** [comment]
@mMoliere Can you give me a record and data to check so I can check that the updates are happening correctly?

**Comment by wsalesky** [comment]
We can modify the zotero2bibl and reimport, let me take a look this morning and get back to you.

**Comment by mMoliere** [comment]
@wsalesky Sure, for the record 75PTS4B6.xml we expect the title "The Oxford Dictionary of the Jewish Religion" in a separate field. And it should then be displayed in Srophé. 
The page number 742 is already in the field `biblScope`, but it is not displayed in Srophé.

For the record EFVRVADD.xml the DOI "https://doi.org/10.2307/1451104" should be in a dedicated field and Srophé should display it after the page numbers (also not displayed now).

**Comment by wsalesky** [comment]
@mMoliere How would you like these new elements to translate to TEI?

**Comment by wsalesky** [comment]
@mMoliere Also, when you say it would "it would also be acceptable if Srophé displayed the formatted bibl element"

Where do you mean? In the bibliography on the individual persons pages: https://manuforma-staging.jalit.org/exist/apps/majlis/person/11 ? In the full page view for the bibl entry: https://manuforma-staging.jalit.org/exist/apps/majlis/bibl/EFVRVADD

Somewhere else? Thanks for the clarification.

**Comment by mMoliere** [comment]
@wsalesky If it would make it easier, it would be acceptable if the formatted bibl element be displayed in the bibliography collapsible and in the full page view instead of the display generated by Srophé – the formatted bibl is already formatted as we want it and it would only be necessary to style sections as cursive text.

e.g. 

```
<bibl type="formatted" subtype="citation"
        resp="https://www.zotero.org/styles/chicago-note-bibliography-17th-edition">Nadia Vidro,
          <title level="a">Aviv barley and calendar diversity among Jews in eleventh-century
          Palestine,</title>
        <title level="m">Journal of Jewish Studies</title> 72, no. 2 (2021): 283–312,
        https://doi.org/https://doi.org/10.18647/3504/jjs-2021.</bibl>
```

**Comment by wsalesky** [comment]
@mMoliere Yes, that would be the easiest solution. I will do that tonight.

**Comment by mMoliere** [comment]
@wsalesky Great then let's do it this way. The two title elements get different styling and in the case of chapters, the page numbers need to be appended.

`<title level="a">`: Quotation marks

`<title level="m">`: Cursive script

Examples:

## record 75PTS4B6: 

```
<bibl type="formatted" subtype="citation"
        resp="https://www.zotero.org/styles/chicago-note-bibliography-17th-edition">David E. Sklare,
          <title level="a">Yeshu‛a Ben Yehudah (11th c.),</title> in <title level="m">The Oxford
          Dictionary of the Jewish Religion</title>, ed. R. J. Zwi Werblowsky and Geoffrey Wigoder
        (New York: Oxford University Press, 1997).</bibl>

**PLUS the contents of biblScope**
```

Displayed as: 

David E. Sklare, "Yeshu‛a Ben Yehudah (11th c.)," in _The Oxford Dictionary of the Jewish Religion_, ed. R. J. Zwi Werblowsky and Geoffrey Wigoder (New York: Oxford University Press, 1997), 742.

## record EFVRVADD

```
<bibl type="formatted" subtype="citation"
        resp="https://www.zotero.org/styles/chicago-note-bibliography-17th-edition">Samuel Abraham
        Poznański, <title level="a">The Karaite Literary Opponents of Saadiah Gaon in the Eleventh
          Century,</title>
        <title level="m">Jewish Quarterly Review</title> 18 (1906): 59–83,
        https://doi.org/https://doi.org/10.2307/1451104.</bibl>
```

Displayed as: 

Samuel Abraham Poznański, "The Karaite Literary Opponents of Saadiah Gaon in the Eleventh Century," _Jewish Quarterly Review_ 18 (1906): 59–83, https://doi.org/https://doi.org/10.2307/1451104.

**Comment by mMoliere** [comment]
@wsalesky Great work, Winona! That helps a lot.
